oke_cluster_engine.tf

```diff
@@ -58,10 +65,13 @@ resource oci_containerengine_node_pool oke_containerengine_node_pool {
     }
   }
   node_source_details {
-    boot_volume_size_in_gbs = "100"
+    boot_volume_size_in_gbs = var.oke_boot_vol_size_gb
```

Comment:
This allows us to configure the size of boot volume - Bigger volume for prod and load test environments.